source/_integrations/weather.template.markdown

```diff
@@ -43,7 +43,7 @@ weather:
 name:
   description: Name to use in the frontend.
   required: true
-  type: string
+  type: template
```

Comment:
I dos not seem the `name` property accepts a template value refering to your PR. Or am I wrong?

Reply:
https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/template_entity.py#L66

Reply:
I did not see cv.ensure_temlate in the schema. So it seem that is added later.